examples/fluent-bit/grafana-loki/README.md

```diff
@@ -0,0 +1,24 @@
+### FireLens Example: Logging to Grafana Loki with Fluent Bit
+
+For documentation on sending your FireLens monitored log data to Grafana Loki, see: [Fluent Bit and FireLens documentation](https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki). Be aware there is a separate Golang output plugin provided by [Grafana](https://grafana.com/docs/loki/latest/clients/fluentbit/) with different configuration options. The Loki data source provides access to Loki, Grafana’s log aggregation system.
+
+It should be noted that this example and Grafana docs show `"enable-ecs-log-metadata":"true"` (which is the default). This option tells FireLens to add ECS Task Metadata keys to logs.  
+
+For all configuration parameters for Fluent Bit Grafana output plugin, see [Fluent Bit](https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki) documentation.
+
+AWS recommends that you store sensitive information, like your Grafana API Key using [secretOptions](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Secret.html) as shown in the example Task Definition. This is optional; it is also valid to simply specify the API Key in options map:
```

Comment:
this output definition is now the same as the one in the task def file... so I think you should remove this... let me know if we can chat briefly on this..